### PR TITLE
V3 updates

### PR DESCRIPTION
**Pull request by tenax002**

### CODE REVIEW

File: examples/place-option-order.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ async function main() {
                                     .set('second', 0)
                                     .unix()
     // Convert Unix timestamp to readable date string for use with Arrow
-    const readableExpiration = getReadableTimestamp(twoWeeksFromNow * 1000) // MM is month, DD is day, and YYYY is year
+    const readableExpiration = "08192022"
```
Comment (tenax002) [inline examples/place-option-order.ts]: Just noticed this is hard coded. Updating to dynamically select the nearest Friday expiration so this test file won't fail in the future.

File: examples/place-option-order.ts [hunk 2]
```diff
@@ -103,10 +104,9 @@ async function main() {
         }
     }
 
+    //TO-DO- Update API to ensure it is generating the correct option chain address
```
Comment (Pdbz199) [inline examples/place-option-order.ts]: I guess we should remove this

File: src/arrow-sdk.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios'
 import { ethers } from 'ethers'
-import moment from 'moment'
+import * as moment from "moment"
```
Comment (Pdbz199) [inline src/arrow-sdk.ts]: lets use single quotes to be consistent

File: examples/place-option-order.ts [hunk 4]
```diff
@@ -19,6 +19,7 @@ import {
     getReadableTimestamp
 } from '../lib/src/arrow-sdk'
 import { ethers } from 'ethers'
+import * as moment from "moment"
```
Comment (Pdbz199) [inline examples/place-option-order.ts]: can we use single quotes here to be the same as other imports?

**Review by Pdbz199 — CHANGES_REQUESTED** [review]
Small comments

**Review by Pdbz199 — APPROVED** [review]

**Pull request merged**